integration-tests/gradle/src/main/resources/basic-java-platform-module/application/src/main/docker/Dockerfile.native

```diff
@@ -1,30 +0,0 @@
-####
-# This Dockerfile is used in order to build a container that runs the Quarkus application in native (no JVM) mode
```

Comment:
Everything else looks straight forward - but why deleting the two Dockerfile(s) ?

Reply:
I had the same question when I looked at the PR and concluded that they were removed because there was no reason for them being there in the first place